lib/jekyll/command.rb

```diff
@@ -73,6 +73,29 @@ def add_build_options(cmd)
           "Fail if errors are present in front matter"
       end
       # rubocop:enable Metrics/MethodLength
+
+      # Run ::process method in a given set of Jekyll::Command subclasses and suggest
+      # re-running the associated command with --trace switch to obtain any additional
+      # information or backtrace regarding the encountered Exception.
+      #
+      # cmd     - the Jekyll::Command to be handled
+      # options - configuration overrides
+      # klass   - an array of Jekyll::Command subclasses associated with the command
+      #
+      # rubocop: disable RescueException
```

Comment:
I wonder if we want to add a comment explaining that we definitely _do_ want to `rescue Exception` here.

I'm just worried that a year from now I might be trying to clean up the code for a new version of Rubocop and forget that this is intentional and not just a bandage.